File: client-sdk/java/sdk/test/java/com/vertexcache/sdk/VertexCacheSDKTest.java
package com.vertexcache.sdk;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class VertexCacheSDKTest {
    @Test
    void testGetMessage() {
        VertexCacheSDK sdk = new VertexCacheSDK();
        assertEquals("VertexCache SDK!", sdk.getMessage());
    }
}

File: client-sdk/kotlin/sdk/src/test/kotlin/com/vertexcache/sdk/VertexCacheSDKTest.kt
package com.vertexcache.sdk

import org.junit.jupiter.api.Assertions.assertEquals
import org.junit.jupiter.api.Test

class VertexCacheSDKTest {

    @Test
    fun `should return correct message`() {
        val sdk = VertexCacheSDK()
        val result = sdk.getMessage()
        assertEquals("VertexCache SDK!", result)
    }
}
